runtime/development/src/lib.rs

```diff
@@ -1116,6 +1118,29 @@ impl pallet_pool_registry::Config for Runtime {
 	type WeightInfo = weights::pallet_pool_registry::WeightInfo<Runtime>;
 }
 
+parameter_types! {
+	pub const MaxPoolFeesPerBucket: u32 = MAX_POOL_FEES_PER_BUCKET;pub const PoolFeesPalletId: PalletId = cfg_types::ids::POOL_FEES_PALLET_ID;
```

Comment:
```suggestion
	pub const MaxPoolFeesPerBucket: u32 = MAX_POOL_FEES_PER_BUCKET;
	pub const PoolFeesPalletId: PalletId = cfg_types::ids::POOL_FEES_PALLET_ID;
```

Reply:
Search + replace failure together with cargo fmt not working inside macro scope :D Great catch